backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -269,98 +192,79 @@ public void addEmail(Email email) {
+                .createMulti()
+                .collect().with(Collectors.joining())
+                .onFailure()
+                .recoverWithItem(templateEx -> {
+                    log.log(Level.WARNING, templateEx, () -> String.format(
+                            "Unable to render template title for application: [%s], eventType: [%s], subscriptionType: [%s].",
+                            item.getAction().getApplication(),
+                            item.getAction().getEventType(),
+                            emailSubscriptionType
+                    ));
+                    return null;
                 });
     }
 
-    @Scheduled(identity = "dailyEmailProcessor", cron = "{email.subscription.daily.cron}")
-    public void processDailyEmail(ScheduledExecution se) {
-        // Only delete on the largest aggregate time frame. Currently daily.
-        processAggregateEmails(se.getScheduledFireTime(), EmailSubscriptionType.DAILY, true).await().indefinitely();
+    private Uni<Tuple3<Email, String, String>> combine(Email email, Uni<String> title, Uni<String> body) {
+        return Uni.combine().all()
+                .unis(
+                        Uni.createFrom().item(email),
+                        title,
+                        body
+                ).asTuple()
+                .onItem().transform(objects -> {
+                    if (objects == null || objects.getItem1() == null || objects.getItem2() == null || objects.getItem3() == null) {
+                        return null;
+                    }
+
+                    return objects;
+                });
     }
 
+    @Incoming("aggregation")
+    public void consumeEmailAggregations(String aggregationsJson) throws JsonProcessingException {
+        List<EmailAggregation> aggregations = objectMapper.readValue(aggregationsJson, new TypeReference<>() { });
+        Uni<List<EmailAggregation>> emailaggregations = Uni.createFrom().item(aggregations);
+        emailaggregations.onItem().transform(aggregationList -> {
+            aggregationList.forEach(emailAggregationItem -> {
+                Action action = new Action();
+                action.setContext(emailAggregationItem.getPayload().getMap());
+                action.setEvents(List.of());
+                action.setAccountId(emailAggregationItem.getAccountId());
+                action.setApplication(emailAggregationItem.getApplicationName());
+                action.setBundle(emailAggregationItem.getBundleName());
+
+                // We don't have a eventtype as this aggregates over multiple event types
+                action.setEventType(null);
+                action.setTimestamp(LocalDateTime.now(ZoneOffset.UTC));
+
+                // We don't have any endpoint (yet) as this aggregates multiple endpoints
+                Notification item = new Notification(action, null);
+
+                sendEmail(item, EmailSubscriptionType.DAILY);
+            });
+            return null;
+        });
+    }
```

Comment:
This (untested code written from GH) should fix the issue.

I added a few comments for the review process but don't commit them 😃 
```suggestion
    @Incoming("aggregation")
    // If the method returns Uni<Void> instead of void, Quarkus will automatically subscribe to the Uni.
    public Uni<Void> consumeEmailAggregations(String aggregationsJson) throws JsonProcessingException {
        List<EmailAggregation> aggregations = objectMapper.readValue(aggregationsJson, new TypeReference<>() { });
        // Each element of the "aggregations" list must be processed as a reactive (Uni) operation because the email won't be sent otherwise, so we need to transform the List into a Multi.
        return Multi.createFrom().iterable(aggregations)
                // Each element of the Multi will be treated as a Uni.
                // Concatenate prevents concurrent access to the Hibernate Reactive session (that's forbidden).
                .onItem().transformToUniAndConcatenate(emailAggregationItem -> {
                    Action action = new Action();
                    action.setContext(emailAggregationItem.getPayload().getMap());
                    action.setEvents(List.of());
                    action.setAccountId(emailAggregationItem.getAccountId());
                    action.setApplication(emailAggregationItem.getApplicationName());
                    action.setBundle(emailAggregationItem.getBundleName());

                    // We don't have a eventtype as this aggregates over multiple event types
                    action.setEventType(null);
                    action.setTimestamp(LocalDateTime.now(ZoneOffset.UTC));

                    // We don't have any endpoint (yet) as this aggregates multiple endpoints
                    Notification item = new Notification(action, null);

                    return sendEmail(item, EmailSubscriptionType.DAILY);
                })
                // The following line will wait for all the Unis above to be resolved and then replace everything with a Uni<Void> value because that's all we need in the end.
                .onItem().ignoreAsUni();
    }
```